riscos_toolbox/mixins/printing.py

```diff
@@ -51,23 +51,23 @@ def print_event(self, code, id_block, event):
         sideways = event.sideways
 
         try:
-            features = swi.swi("PDriver_Info",";...I")
+            features = swi.swi("PDriver_Info", ";...I")
         except:
             raise RuntimeError("Printer driver not loaded.")
 
-        job = swi.swi("OS_Find","Is;I",0x83,"printer:")
+        job = swi.swi("OS_Find", "Is;I", 0x83, "printer:")
```

Comment:
A SWI error might occur as well, so you might want to trap this and return an appropriate error message.

Reply:
Moved to issue #38